setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 
 
-install_requires = ['PyMySQL>=0.9,<=0.9.3']
+install_requires = ['PyMySQL>=0.9,<1.1']
```

Comment:
based on https://github.com/PyMySQL/PyMySQL/issues/882 it seems that PyMySQL does not use semantic versioning, so that should probably be pinned to `<1.0.3` instead of `<1.1`.